variables.tf

```
@@ -1,25 +1,30 @@
 variable "awp_cloud_account_id" {
-    description = "CLOUDGUARD_ACCOUNT_ID or EXTERNAL_AWS_ACCOUNT_NUMBER"
-    type        = string
+  description = "CLOUDGUARD_ACCOUNT_ID or EXTERNAL_AWS_ACCOUNT_NUMBER"
+  type        = string
+}
+
+variable "awp_centralized_cloud_account_id" {
+  description = "centralized account EXTERNAL_AWS_ACCOUNT_NUMBER or EXTERNAL_AWS_ACCOUNT_NUMBER"
```

EXTERNAL_AWS_ACCOUNT_NUMBER or EXTERNAL_AWS_ACCOUNT_NUMBER ???
```suggestion
  description = "centralized account CLOUDGUARD_ACCOUNT_ID or EXTERNAL_AWS_ACCOUNT_NUMBER"
```

---

done thanks